app/controllers/public_reviews_controller.rb

```diff
@@ -0,0 +1,36 @@
+class PublicReviewsController < ApplicationController
+
+  def new_reviewer
+    @person = Person.new
+  end
+
+  def create_reviewer
+    if
+      @person = Person.find(params[:person_id].where(Person.email == @person.email))
```

Comment:
Przypisanie `@person` trzeba przenieś przed if.
Poza tym to zapytanie powinno wyglądać następująco:
```ruby
@person = Person.where(email: params[:person][:email]).first
if @person
...
```